src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
```

Comment:
Use decompose structure where is possible 
```suggestion
export const TodoInfo: React.FC<Props> = ({ todo: { id, title, user } }) => (
```